spec/controllers/idv/personal_key_controller_spec.rb

```diff
@@ -52,81 +54,175 @@ def assert_personal_key_generated_for_profiles(*profile_pii_pairs)
       idv_session.address_verification_mechanism = 'phone'
       idv_session.user_phone_confirmation = true
       idv_session.vendor_phone_confirmation = true
+      idv_session.gpo_code_verified = false
     when 'gpo'
       idv_session.address_verification_mechanism = 'gpo'
       idv_session.user_phone_confirmation = false
       idv_session.vendor_phone_confirmation = false
+      idv_session.gpo_code_verified = true
+    when nil
+      idv_session.address_verification_mechanism = nil
+      idv_session.user_phone_confirmation = nil
+      idv_session.vendor_phone_confirmation = nil
```

Comment:
This setup needs to be moved to FlowPolicyHelper, but maybe that's a separate PR.

Reply:
I updated _most_ of this to use FlowPolicyHelper, I don't think gpo_code_verified is doable via FlowPolicy yet.